gwu-libraries/sfm-ui
# T1011 datetime picker

To test:
1. Do an export and select item start / item end dates (and times).
2. Do an export and select harvest start / harvest end dates (and times).
3. Create a collection and set an end date on the collecting. 

The dates should be reflected in the export.
